stage_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Get names of all cronjobs, if not found any ignore the error message
+# to not let k8s raise an error with an exit code indicating success
+CRONJOBS=$(oc get cronjobs 2>/dev/null | awk 'NR>1 {print $1}')
+
+if [ -z "$CRONJOBS" ]; then
+    echo "ERROR: no cronjobs found"
+    exit 1
+fi
+
+for CRONJOB in $CRONJOBS; do
+    SUCCESS=$(oc get job -l "pod=${CRONJOB}" -o jsonpath='{.items[].status.succeeded}{"\n"}')
```

Comment:
On other note, it would be good to check the image that these cron jobs use and match those with what the controller was configured with. That would check that the operator have changed them properly. We can do this as a followup.

Reply:
Do you mean the image of floorist or the operator itself? Just to clarify.

Reply:
floorist image on these (cron)jobs. The main reason for the operator is that these images (in jobs) are managed centrally. The check would ensure that the operator has changed the resources accordingly.